src/clr-angular/accordion/providers/accordion.service.spec.ts

```diff
@@ -0,0 +1,61 @@
+/*
+* Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+* This software is released under MIT license.
+* The full license information can be found in LICENSE in the root directory of this project.
+*/
+
+import { AccordionService } from './accordion.service';
+import { take } from 'rxjs/operators';
+import { AccordionStrategy } from '../enums/accordion-strategy.enum';
+
+describe('AccordionService', () => {
+  let accordionService: AccordionService;
+  const panel1Id = '0';
+  const panel2Id = '1';
+
+  beforeEach(() => {
+    accordionService = new AccordionService();
+    accordionService.addPanel(panel1Id);
+    accordionService.addPanel(panel2Id);
+    accordionService.updatePanelOrder([panel1Id, panel2Id]);
+  });
+
+  it('should get updates of an individual panel change', () => {
+    accordionService
+      .getPanelChanges(panel1Id)
+      .pipe(take(1))
+      .subscribe(panel => expect(panel.id).toBe(panel1Id));
+  });
+
+  it('should update of panel changes when toggling to new panel', () => {
+    accordionService.togglePanel(panel2Id);
+    accordionService
+      .getPanelChanges(panel1Id)
+      .pipe(take(1))
+      .subscribe(panel => expect(panel.open).toBe(false));
+  });
+
+  it('should notify of panel changes when panel order has changed', () => {
+    accordionService.updatePanelOrder([panel2Id, panel1Id]);
+    accordionService
+      .getPanelChanges(panel1Id)
+      .pipe(take(1))
+      .subscribe(panel => expect(panel.index).toBe(1));
+  });
+
+  it('should allow component to set the active accordion strategy', () => {
```

Comment:
Should there be a test that verify's the default strategy is what consumers get out of the box?

Reply:
Added additional check in the accordion component spec, the component determines which strategy and passes that to the service.